src/middleware/middleware.ts

```diff
@@ -26,6 +26,7 @@ export function middleware(
     }
 
     debugWebhooks(`ignored: ${request.method} ${request.url}`);
+    state.log.debug(`ignored: ${request.method} ${request.url}`);
```

Comment:
I wonder that if `state.log` is set, should we still call `debugWebhooks`? We could introduce the `{ log }` option without a breaking change that way. With the next breaking version, we will simply remove the `debugWebhooks` altogether.

In either way, we should add a test that captures the stdout output

Reply:
Good idea, I've revised it to not call `debugWebhooks()` if the log option is set and added a basic test. What do you think?

Reply:
Looks good 👍🏼 

Could you adjust the test so that they don't log out anything, instead pass a mock and add an assertion that the expected output was logged?

<img width="608" alt="image" src="https://user-images.githubusercontent.com/39992/107689725-0d40ac80-6c5e-11eb-9d12-c5e82b68c3a2.png">

Reply:
Done, I've revised the tests so they don't output anything and instead check the output is what's expected.